packages/govuk-frontend/src/govuk/settings/_colours-organisations.scss

```
@@ -2,19 +2,214 @@
+/// organisation list and colour selection.
+///
+/// Should be removed in 6.0.
+///
+/// @type Boolean
+/// @access public
+
+$govuk-new-organisation-colours: false !default;
+
+/// Current organisation colours
 ///
 /// @type Map
 ///
 /// @prop $organisation.colour - Colour for the given `$organisation`
-/// @prop $organisation.colour-websafe - Websafe colour for the given
-///   `$organisation`. We use `websafe` to mean strong enough contrast against
-///   white to be used for copy and meet the AAA (large text) and AA (smaller
+/// @prop $organisation.contrast-safe - 'Contrast safe' colour for the given
+///   `$organisation`. This colour has at least a 4.5:1 contrast against
+///   white, to be used for copy and meet the AAA (large text) and AA (smaller
 ///   copy) WCAG guidelines.
+/// @prop $organisation.deprecation-message - Marks this organisation as
+///   deprecated (usually because it has ceased to exist). Value is the
+///   deprecation message output to the terminal when compiling.
 ///
-/// @access public
+/// @access private
 
-$govuk-colours-organisations: (
+$_govuk-organisation-colours: (
+  "attorney-generals-office": (
+    colour: #a91c8e
+  ),
+  "cabinet-office": (
+    colour: #0056b8
+  ),
+  "civil-service": (
+    colour: #b2292e
+  ),
+  "department-for-business-trade": (
+    colour: #e52d13,
+    contrast-safe: #e02c13
+  ),
+  "department-for-business-energy-industrial-strategy": (
+    colour: #003479,
+    deprecation-message:
+      "`department-for-business-energy-industrial-strategy` was dissolved in 2023. It was replaced by `department-for-business-trade`, `department-for-energy-security-net-zero` and `department-science-innovation-technology`."
+  ),
+  "department-for-business-innovation-skills": (
+    colour: #003479,
+    deprecation-message:
+      "`department-for-business-innovation-skills` became `department-for-business-energy-industrial-strategy` in 2016. As of 2023, it is equivalent to `department-for-business-trade` or `department-science-innovation-technology`."
+  ),
+  "department-for-communities-local-government": (
+    colour: #009999,
+    contrast-safe: #008080,
+    deprecation-message:
+      "`department-for-communities-local-government` became `ministry-of-housing-communities-local-government` in 2018. As of 2021, it is equivalent to `department-for-levelling-up-housing-communities`."
+  ),
+  "department-for-culture-media-sport": (
+    colour: #d6006e
+  ),
+  "department-for-digital-culture-media-sport": (
+    colour: #d40072,
+    deprecation-message:
+      "`department-for-digital-culture-media-sport` became `department-for-culture-media-sport` in 2023."
+  ),
+  "department-for-education": (
+    colour: #003764
+  ),
+  "department-for-energy-security-net-zero": (
+    colour: #00a33b,
+    contrast-safe: #00852f
+  ),
+  "department-for-environment-food-rural-affairs": (
+    colour: #00af43,
+    contrast-safe: #008733
+  ),
```

Hey @querkmachine 👋

I've done some digging and your suggested `contrast-safe` colour is almost spot on, but mind if I provide a tiny tweak from the Defra brand team?

> Please use `#008531`. It achieves the required WCAG AA standard (and AAA for everything except normal text) and is noticeably closer to actual Defra green (`#00A33B`) than the proposed darker green.

Although their suggestion shows we're still using `#00a33b` "Defra green" which is now in the legacy colours in this PR

Are you able to share your Design102 guidance showing where the newly suggested `#00af43` came from? Would be good to join the dots

Thanks 🙌

```suggestion
  "department-for-environment-food-rural-affairs": (
    colour: #00af43,
    contrast-safe: #008531
  ),
```

---

`#00af43` came from a draft update to the HMG Identity Guidelines that was shared with me last year. As far as I can tell, that update still hasn't been released, so I don't have anything I can forward on that front. 

Curiously, the most recent release version of the HMG Identity Guidelines (from 2022) uses `#01ab54` as Defra's departmental colour, which is different again.

In the past, individuals from departments have suggested lots of colours for the department brand or contrast-safe variants, some of them wildly deviating from the brand colour or the HMG Identity Guidelines colour. 

We don't want to act as the arbiters of what is a correct or incorrect colour. Part of the intent of this PR is to source department colours from the best-source-of-truth (the HMG Identity Guidelines) and use a consistent, programmatic methodology to create the contrast-safe variant (and any other variants that may exist in future).

That's not to say that the methodology used cannot be altered, but if we were to change the methodology for Defra, we'd also want to change it for every other department. 

Accepting a one-off change would be anthithetical to our goals here, and keep us stuck in the mire of piecemeal and haphazardly updated organisation colours that we've had for the last several years.

---

>Are you able to share your Design102 guidance showing where the newly suggested #00af43 came from? Would be good to join the dots

Thanks @querkmachine, can you help me join the dots?

I'd like to share the ~Design102~ HMG Identity Guidelines (2022) with the Defra brand team. At the moment they've pointed out the colours in this PR are incorrect, but I'm just the messenger

I'll message on Slack